lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -0,0 +1,21 @@
+;(function (define) {
```

Comment:
Thanks - I was just asking because I haven't seen the "_factory" naming pattern for Backbone modules before.  I was also thrown off by the fact that the module instantiates a program list view, but also another view, which seemed counter-intuitive.  I guess what I missed is that this module pertains to the program list page, rather than the program list view.